azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -830,3 +881,13 @@ func resourceAzureRMCosmosDBAccountGeoLocationHash(v interface{}) int {
 
 	return hashcode.String(buf.String())
 }
+
+func resourceAzureRMCosmosDBAccountCapabilitiesHash(v interface{}) int {
```

Comment:
_minor_ we don't technically need this hash function, since we should be able to use the default one